newIDE/app/src/UI/FlatButton.js

```diff
@@ -22,22 +24,31 @@ type Props = {|
     flexShrink?: 0,
   |},
   target?: '_blank',
+  exceptionalTooltipForToolbar?: React.Node,
 |};
 
 /**
  * A "flat" button based on Material-UI button.
  */
 export default class FlatButton extends React.Component<Props, {||}> {
   render() {
-    const { label, primary, icon, keyboardFocused, ...otherProps } = this.props;
+    const {
+      label,
+      primary,
+      icon,
+      keyboardFocused,
+      exceptionalTooltipForToolbar,
```

Comment:
This `exceptional` props is a bit scary. Can you add documentation to explain why it is here?